main.cpp

```diff
@@ -478,14 +452,34 @@ IntType& IntType::powInternal( int value )
     IntType it ( 34 );
     DoubleType pi( 3.14 );
 
-    std::cout << "The result of FloatType^3 divided by IntType is: " << ft.multiply( static_cast<float>(ft) ).multiply( static_cast<float>(ft) ).divide( static_cast<float>(it) ) << std::endl;
-    std::cout << "The result of DoubleType times 3 plus IntType is : " << dt.multiply( static_cast<double>(3) ).add( static_cast<double>(it) ) << std::endl;
-    std::cout << "The result of IntType divided by 3.14 multiplied by DoubleType minus FloatType is: " << it.divide( static_cast<int>(pi) ).multiply( static_cast<int>(dt) ).subtract( static_cast<int>(ft) ) << std::endl;
+    ft*=ft;
+    ft*=ft;
+    ft/=it;
+    std::cout << "The result of FloatType^3 divided by IntType is: " << ft << std::endl;
+
+    dt*=3;
+    dt+=it;
+    std::cout << "The result of DoubleType times 3 plus IntType is : " << dt << std::endl;
+
+    it/=static_cast<int>(pi);
+    it*=static_cast<int>(dt);
+    it-=static_cast<int>(ft);
+    std::cout << "The result of IntType divided by 3.14 multiplied by DoubleType minus FloatType is: " << it << std::endl;
     std::cout << "An operation followed by attempts to divide by 0, which are ignored and warns user: " << std::endl;
-    std::cout << it.multiply(static_cast<int>(it)).divide(static_cast<int>(0)).divide(static_cast<int>(0.0f)).divide(0.0) << std::endl;
+
+    it*=it;
+    it/=0;
+    it/=0.0f;
+    it/=0.0;
+    std::cout << it << std::endl;
     
-    std::cout << "FloatType x IntType  =  " << it.multiply( static_cast<int>(ft) ) << std::endl;
-    std::cout << "(IntType + DoubleType + FloatType) x 24 = " << it.add( static_cast<int>(dt) ).add( static_cast<int>(ft) ).multiply( static_cast<int>(24) ) << std::endl;
+    it*=static_cast<int>(ft);
+    std::cout << "FloatType x IntType  =  " << it << std::endl;
+
+    it+=static_cast<int>(dt);
+    it+=static_cast<int>(ft);
+    it*=24;
+    std::cout << "(IntType + DoubleType + FloatType) x 24 = " << it << std::endl;
```

Comment:
Readability is very important.  
Add spaces around your operators!
- [ ] revise